src/pages/Designer.js

```diff
@@ -1064,8 +1127,10 @@ const Designer = ({ update, viewMode = false }) => {
                     ...logManagementData,
                 };
             if (Node.id.startsWith('UI')) {
-                if (Node.data.applicationFramework !== 'docusaurus') Node.data.applicationFramework = Node.data.clientFramework;
-                else Node.data.packageName = 'docs';
+                if (Node.data.applicationFramework !== 'docusaurus') {
```

Comment:
try always write the positive condition --> improves user readability
for example this can be:-
if (Node.data.applicationFramework === 'docusaurus') {
Node.data.packageName = 'docs';
} else {
Node.data.packageName = 'ui';
}

and enclose the else block as well.